ui/app/clinical/displaycontrols/investigationresults/views/investigationTableRow.html

```diff
@@ -6,14 +6,10 @@
     <td class="testUnderPanel" bo-class="{'has-uploaded-file': test.uploadedFileName}">
         <span ng-if="!test.uploadedFileName" bo-text="::getLocaleSpecificNameForTest(test)"></span>
         <span ng-if="test.labReportUrl">
-            <a class="uploaded-file"  href="{{test.labReportUrl}}" stop-event-propagation="click" target="_blank">{{test.preferredNameByLocale}}
-                <span bo-show="test.hasRange" class="range">
-                    <span bo-text="getFormattedRange(test)"></span>
```

Comment:
@mohan-13 are we still be able to see the formatted range when the file is uploaded for a test as mentioned in the below card
https://bahmni.atlassian.net/browse/BAH-2306

Reply:
> @mohan-13 are we still be able to see the formatted range when the file is uploaded for a test as mentioned in the below card with PR changes ? https://bahmni.atlassian.net/browse/BAH-2306

Yes Bindu. The range is still visible even if we upload a file.
<img width="635" alt="Screenshot 2023-03-07 at 10 59 04 AM" src="https://user-images.githubusercontent.com/31698165/223328992-c0344866-27e8-4525-82f9-bb30ef67583c.png">